charts/portworx/v1.0.9/templates/portworx-ds.yaml

```diff
@@ -0,0 +1,389 @@
+{{/* Setting defaults if they are omitted. */}}
```

Comment:
looks like rancher is still missing the portworx-api ds and service that you recently added to the portworx/helm chart. Let's add that here. It's needed for certain stork features like DR.